fcrepo-exts/fcrepo-docker
# FCREPO-3193: Update README, add travis pipeline

JIRA Ticket: https://jira.lyrasis.org/browse/FCREPO-3193

# What does this Pull Request do?

- Update README.md with instructions
- Add travis pipeline

This PR resolves the remaining open tasks in https://jira.lyrasis.org/browse/FCREPO-3193.

# How should this be tested?

1. Checkout
2. `cd fcrepo-docker`
2. Run `build.sh /path/to/locally/built/fcrepo-webapp-6.0.0-SNAPSHOT.war`
3. Run `docker run -p8080:8080 --name=fcrepo fcrepo/fcrepo` to start a Docker container and make it available on port 8080.

Or

1. Enable Travis pipeline

--> The docker image should be built within the pipeline, but will not be pushed.

# Interested parties

@dbernstein 
@awoods
